### ISSUE TITLE
PHP8 compatibility

### ISSUE THREAD

**Issue by lucaguindani** [unil-lettres/dilps-tiresias#79]
Multiple incompatibilities with php dependencies. Waiting compatible versions to bump incompatible dependencies.

```
  Problem 1
    - Root composer.json requires php ^7.4 but your php version (8.0.2) does not satisfy that requirement.
  Problem 2
    - dflydev/fig-cookies is locked to version v2.0.3 and an update of this package was not requested.
    - dflydev/fig-cookies v2.0.3 requires php ^7.2 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 3
    - doctrine/orm is locked to version 2.7.5 and an update of this package was not requested.
    - doctrine/orm 2.7.5 requires php ^7.1 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 4
    - ecodev/felix is locked to version 5.2.3 and an update of this package was not requested.
    - ecodev/felix 5.2.3 requires php ^7.2 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 5
    - ecodev/graphql-doctrine is locked to version 7.1.2 and an update of this package was not requested.
    - ecodev/graphql-doctrine 7.1.2 requires php ^7.2 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 6
    - laminas/laminas-json is locked to version 3.1.2 and an update of this package was not requested.
    - laminas/laminas-json 3.1.2 requires php ^5.6 || ^7.0 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 7
    - laminas/laminas-loader is locked to version 2.6.1 and an update of this package was not requested.
    - laminas/laminas-loader 2.6.1 requires php ^5.6 || ^7.0 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 8
    - laminas/laminas-mime is locked to version 2.7.4 and an update of this package was not requested.
    - laminas/laminas-mime 2.7.4 requires php ^5.6 || ^7.0 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 9
    - laminas/laminas-permissions-acl is locked to version 2.7.2 and an update of this package was not requested.
    - laminas/laminas-permissions-acl 2.7.2 requires php ^5.6 || ^7.0 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 10
    - mezzio/mezzio is locked to version 3.2.2 and an update of this package was not requested.
    - mezzio/mezzio 3.2.2 requires php ^7.1 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 11
    - mezzio/mezzio-laminasviewrenderer is locked to version 2.2.0 and an update of this package was not requested.
    - mezzio/mezzio-laminasviewrenderer 2.2.0 requires php ^7.1 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 12
    - mezzio/mezzio-template is locked to version 2.0.1 and an update of this package was not requested.
    - mezzio/mezzio-template 2.0.1 requires php ^7.1 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 13
    - roave/psr-container-doctrine is locked to version 2.2.0 and an update of this package was not requested.
    - roave/psr-container-doctrine 2.2.0 requires php >=7.3.0,<7.5.0 -> your php version (8.0.2) does not satisfy that requirement.
  Problem 14
    - doctrine/orm 2.7.5 requires php ^7.1 -> your php version (8.0.2) does not satisfy that requirement.
    - creof/doctrine2-spatial 1.2.0 requires doctrine/orm >=2.3 -> satisfiable by doctrine/orm[2.7.5].
    - creof/doctrine2-spatial is locked to version 1.2.0 and an update of this package was not requested.
 ```

**Comment by lucaguindani** [comment]
Once compatible the following line can be removed.
https://github.com/unil-lettres/dilps-tiresias/blob/96b482a64bef6c0bae79eb30c5fc6272e4064f84/.github/workflows/ci.yml#L20

**Comment by lucaguindani** [comment]
Solved by 7cbae96781670db8657286baf51bbeecc666d744
